api/src/test/java/org/openmrs/module/locationbasedaccess/aop/UserSearchAdviserTest.java

```diff
@@ -0,0 +1,118 @@
+package org.openmrs.module.locationbasedaccess.aop;
+
+import java.util.List;
+import org.junit.Test;
+import static org.junit.Assert.*;

```

Comment:
We do not recommend using wild card imports https://wiki.openmrs.org/display/docs/Java+Conventions#JavaConventions-Donotusewildcardimports

Reply:
@dkayiwa Oh I forgot to change that, had written it earlier to make the IDE happy.